Makefile

```diff
@@ -30,7 +30,7 @@ else
 
 # Command generalization
 	RM_CMD=rm -rf ./**/__pycache__;rm -rf __pycache__
-	EXPORT_CMD=$(PY_CONDA) -m pip freeze | grep -v "@ file" >requirements.txt;
+	EXPORT_CMD=$(PY_CONDA) -m pip list --format=freeze > requirements.txt;
```

Comment:
this could fit better in the DRL branch alongside the splitting of the requirements files no ?

Reply:
plus the change needs to be done for the windows part as well.

Reply:
Agreed, but I am not sure if we need the split requirements files anymore, thus why I put it here.

Reply:
I have deleted the wexport part and unified the` make export` for all systems.  @m-aspro can you test it and tell me if the new command works for Windows?